priv/repo/migrations/20200921050912_disabling_hyperlinks.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Glimesh.Repo.Migrations.DisablingHyperlinks do
+  use Ecto.Migration
+
+  def change do
+    alter table(:channels) do
+      add :disable_hyperlinks, :boolean, default: false
+      add :block_links, :boolean, default: false
```

Comment:
I feel disabling of links being parsed should be done on user side and not channel as channels should only have the option to allow or disallow hyperlinks being posted.